src/server/apis.js

```diff
@@ -12,3 +12,7 @@ fs.readFile(__dirname + '/weather-key', (err, data) => {
 export function getWeatherUrl(lat, lng){
   return `http://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lng}&appid=${weatherKey}`;
 }
+
+export function getStockUrl(nse) {
+  return `http://dev.markitondemand.com/MODApis/Api/v2/Quote/jsonp?symbol=${nse}`;
+}
```

Comment:
This actually returns `jsonp`... which I have never heard of before, and apparently it works great (or something?) for loading into scripts. However if you want to make it work with your current logic just load `json`
change `http://dev.markitondemand.com/MODApis/Api/v2/Quote/jsonp?symbol=${nse}` to
**`http://dev.markitondemand.com/MODApis/Api/v2/Quote/json?symbol=${nse}`**